### PR TITLE
[Keymap] Add MOC keymap for meishi2

### PR DESCRIPTION
**Pull request by ngs**
## Description

Keymap to control [MOC] music player with Meishi2

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

[MOC]: http://moc.daper.net/

### CODE REVIEW

File: keyboards/meishi2/keymaps/moc/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 Biacco42
```
Comment (spidey3) [inline keyboards/meishi2/keymaps/moc/config.h:1]: This should refer to your actual name, not just to a user id.  Also, you might want to update the year.

File: keyboards/meishi2/keymaps/moc/keymap.c [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 Biacco42
```
Comment (spidey3) [inline keyboards/meishi2/keymaps/moc/keymap.c:1]: Same here; it needs a full name, and the correct year.

File: keyboards/meishi2/keymaps/moc/keymap.c [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    _DEFAULT,
+    _RAISE
+};
+
+#define PRO_MICRO_LED_TX D5
+#define RAISE MO(_RAISE)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_DEFAULT] = LAYOUT( /* Base */
+    RAISE, KC_B, KC_N, KC_SPC \
```
Comment (spidey3) [inline keyboards/meishi2/keymaps/moc/keymap.c:28]: No backslash needed at the end of this line.
```suggestion
    RAISE, KC_B, KC_N, KC_SPC
```

File: keyboards/meishi2/keymaps/moc/keymap.c [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  ),
+  [_RAISE] = LAYOUT( /* Raise */
+    _______, KC_LEFT, KC_RGHT, LSFT(KC_S) \
```
Comment (spidey3) [inline keyboards/meishi2/keymaps/moc/keymap.c:31]:
```suggestion
    _______, KC_LEFT, KC_RGHT, LSFT(KC_S)
```

**Review by spidey3 — CHANGES_REQUESTED** [review]

**Review by spidey3 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**